Lib/typing.py

```diff
@@ -1370,6 +1370,14 @@ class _ProtocolMeta(ABCMeta):
     def __instancecheck__(cls, instance):
```

Comment:
Wait, I have a question, do we need to do the same for subclasscheck dunder as well? My data model methods are a little rusty so I might be wrong here.

Reply:
^ Please ignore this, it has been covered by ``__init_subclass__``and ``_proto_hook``!

Reply:
Great, thanks for pointing this out)